bindings/python/src/metadata.rs

```diff
@@ -90,6 +90,11 @@ impl Metadata {
     pub fn mode(&self) -> EntryMode {
         EntryMode(self.0.mode())
     }
+    /// Last modified time
+    #[getter]
+    pub fn last_modified(&self) -> Option<String> {
```

Comment:
It's better to expose as python's datetime class.

Reply:
cool that will be better will change this Sunday